install.sh

```diff
@@ -334,6 +335,10 @@ python -m pip install wheel
 python -m pip install --extra-index-url https://pypi.chia.net/simple/ miniupnpc==2.2.2
 python -m pip install ${EDITABLE} ."${EXTRAS}" --extra-index-url https://pypi.chia.net/simple/
 
+if [ -n "$BLSPY_STUBS" ]; then
+python -m pip install ${EDITABLE} ./blspy-stubs
+fi
+
```

Comment:
If this is truly temporary then I guess we can let it stay separate, but the fewer pip calls the better it can handle itself.  I know we already have two and that's not good.

Reply:
I have a PR in the works that replaces blspy, but it's not ready yet.